app/api/discord/route.ts

```diff
@@ -0,0 +1,26 @@
+import { httpClient } from '@/lib/httpClient'
+import { NextRequest, NextResponse } from 'next/server'
+
+export async function POST(request: NextRequest) {
+  try {
+    const data = await request.json()
+    if (!data.message) {
+      console.error('message not provided for discord notification')
+      return new NextResponse('message not provided for discord notification', {
+        status: 500,
+      })
+    }
+    // sending the message through webhook url
+    await httpClient.post(`${process.env.TEST_WEBHOOK}`, {
```

Comment:
**nitpick:** No need to create API route for that. Such notifications should be called server side, so you can create something like `/lib/discord.ts` where you will have POST to webhook url. Then just use this method when specific events happen, but let's do it server side only